apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/Admin.cs

```diff
@@ -53,5 +53,96 @@ public AdminTests(TestDatabaseFixture testFixture, ITestOutputHelper outputHelpe
                 await dropCommand.ExecuteNonQueryAsync().ConfigureAwait(false);
             }
         }
+
+        [Fact]
+        public async Task CreateWithExtrasDrop()
```

Comment:
Does "WithExtras" here refer to passing multiple commands into `CreateDdlCommand`?

Reply:
yep